src/platform/packages/private/kbn-mock-idp-utils/src/constants.ts

```diff
@@ -17,6 +17,9 @@ export const MOCK_IDP_UIAM_SHARED_SECRET = 'Dw7eRt5yU2iO9pL3aS4dF6gH8jK0lZ1xC2vB
 export const MOCK_IDP_UIAM_ORGANIZATION_ID = '1234567890';
 export const MOCK_IDP_UIAM_PROJECT_ID = 'abcde1234567890';
 
+export const MOCK_IDP_UIAM_SERVICE_INTERNAL_URL = 'http://uiam-cosmosdb:8090';
```

Comment:
__note:__ since both ES and UIAM service are running within the same Docker network they can use this internal addressing. Kibana should still rely on localhost/127.0.0.1